datafusion/core/src/dataframe/mod.rs

```diff
@@ -244,6 +244,30 @@ impl DataFrame {
             .collect();
         self.select(expr)
     }
+    /// Project arbitrary expression strings into a new `DataFrame`.
+    ///
+    /// The output `DataFrame` has one column for each element in `exprs`.
+    ///
+    /// # Example
+    /// ```
+    /// # use datafusion::prelude::*;
+    /// # use datafusion::error::Result;
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<()> {
+    /// let ctx = SessionContext::new();
+    /// let df = ctx.read_csv("tests/data/example.csv", CsvReadOptions::new()).await?;
+    /// let df = df.select_exprs(&["a * b", "c"])?;
+    /// # Ok(())
```

Comment:
why do we have commented out parts of example? 🤔 
and I found it would be really nice to have an expected answer in example. Like what exactly `select_exprs` returns

Reply:
not sure what do you mean @comphead, I just copied doc from `select_columns` and adapt it for this interface (description, method name and args)

Reply:
alright, its probably what we can do next. If one read the `select_expr` its great to have a usage but would be nice to see what the method returns

Reply:
I get you, will change

Reply:
Filed https://github.com/apache/datafusion/issues/12527